### PR TITLE
Prepare environment

### PR DESCRIPTION
**Pull request by fivetran-dmitryboykov**
Development instructions.

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Preparation
+
+## Install extensions
+   - [Bazel](https://marketplace.visualstudio.com/items?itemName=BazelBuild.vscode-bazel)
```
Comment (pgrivachev) [inline DEVELOPMENT.md:4]: Do we actually use it for any task now?

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+```
+bazel build //zetasql/parser/... --features=-supports_dynamic_linker
+```
+
+## Run tests for one package
+```
+bazel test //zetasql/parser/... --features=-supports_dynamic_linker
+```
+
+## Build and debug tests
```
Comment (pgrivachev) [inline DEVELOPMENT.md:23]: This part is not clear. If I run the following commands what will happen? Debug will started? I think we should have more description here.

File: .vscode/launch.json [hunk 3]
```diff
@@ -5,17 +5,26 @@
...
       "args": ["select 1 + 1;"],
       "sourceMap": {
         ".": "${workspaceFolder}"
-      },      
-      // "stopOnEntry": true,
-      "preRunCommands": ["breakpoint set -f ${workspaceFolder}/zetasql/tools/execute_query/execute_query.cc -l 95", "breakpoint list --verbose"]
+      }
+    },
+    {
+      "name": "Execute test",
```
Comment (pgrivachev) [inline .vscode/launch.json:18]: Did you manage how to find sources?

**Review by pgrivachev — COMMENTED** [review]
Could you please elaborate on how to debug code?

File: DEVELOPMENT.md (hunk 1 above)
Comment (fivetran-dmitryboykov) [reply DEVELOPMENT.md:4]: Yes, we will need to review packages dependencies. This extension make bazel files more readable.

File: .vscode/launch.json (hunk 3 above)
Comment (fivetran-dmitryboykov) [reply .vscode/launch.json:18]: No, we just need to build package to test with `-c dbg` key.

**Review by pgrivachev — APPROVED** [review]
Looks good.

**Pull request merged**